### PR TITLE
new webservice USERNAME_EXIST - refs BT#16537

### PR DESCRIPTION
**Pull request by sebastiendu**

### CODE REVIEW

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- main/webservices/api/tests/UsernameExistTest.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4717817&bid=15753330)

File: main/webservices/api/tests/UsernameExistTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
```
Comment (jmontoyaa) [inline main/webservices/api/tests/UsernameExistTest.php]: Codacy Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4717817)

Comment (sebastiendu) [reply main/webservices/api/tests/UsernameExistTest.php]: I do not understand this, I guess the test has a bug.

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- main/webservices/api/tests/UsernameExistTest.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4717817&bid=15753330)

File: main/webservices/api/tests/UsernameExistTest.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
```
Comment (jmontoyaa) [inline main/webservices/api/tests/UsernameExistTest.php]: Codacy Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4717817)

**Pull request merged**